.github/workflows/documentation.yml

```diff
@@ -0,0 +1,23 @@
+name: Generate documentation
+
+on:
+  push:
+    branches: [ docs ]
```

Comment:
Do you want to change this back to `master`?  Or do you plan to keep documentation on the `docs` branch from now on?  Or maybe just use both `master` and `docs`?

Reply:
I plan to switch it back to `master` after merging.